### ISSUE TITLE
How to parse a pdf file and get the text

### ISSUE THREAD

**Issue by infinite-Joy** [J-F-Liu/lopdf#48]

**Comment by J-F-Liu** [comment]
See [`extract_text` method](https://github.com/J-F-Liu/lopdf/blob/master/src/processor.rs#L153-L194), this feature is not complete, need to support CMap and CIDFont.

**Comment by jrmuizel** [comment]
A more sophisticated text extraction library built on top of lopdf is here: https://github.com/jrmuizel/pdf-extract

**Comment by Qingluan** [comment]
> See [`extract_text` method](https://github.com/J-F-Liu/lopdf/blob/master/src/processor.rs#L153-L194), this feature is not complete, need to support CMap and CIDFont.

yes ，it more easy to use "exteact_text" . but i do not know why extract text so slow , even more slow then PyPDF2; Is there some way faster? thanks
